test/index.html

```diff
@@ -10,14 +10,14 @@
 
 <body>
   <script>
-    WCT._config.environmentScripts.push('webcomponentsjs/webcomponents.js');
+    WCT._config.environmentScripts.push('webcomponentsjs/webcomponents-lite.js');
 
     WCT.loadSuites([
       'upload.html',
       'adding-files.html',
       'file-list.html'
     ].reduce(function(suites, suite) {
-      return suites.concat([suite, suite + '?dom=shadow']);
+      return suites.concat([suite, suite + '?wc-shadydom=true&dom=shadow']);
```

Comment:
Yeah, that might be a confusing, but the idea is to use `dom=shadow` when running tests in Polymer v1 to enable shadow and `wc-shadydom=true` to enable shady when running tests in Polymer v2.